pythonforandroid/bootstraps/pygame/__init__.py

```diff
@@ -40,21 +43,14 @@ def run_distribute(self):
             info('Copying python distribution')
             hostpython = sh.Command(self.ctx.hostpython)
             # AND: This *doesn't* need to be in arm env?
-            shprint(hostpython, '-OO', '-m', 'compileall', self.ctx.get_python_install_dir())
+            shprint(hostpython, '-OO', '-m', 'compileall', self.ctx.get_python_install_dir(),
+                    _tail=10, _filterout="^Listing", _critical=True)
```

Comment:
Do you have a reason for limiting to the last 10 lines? If we're passing the output through to the user, don't we want to pass all of it (or, if debug is not enabled, none of it)?